### ISSUE TITLE
Dictionaries and Auto-start

### ISSUE THREAD

**Issue by psychodelicwasabi** [birtles/rikaichamp#41]
My small brain don't seem to be able to make a pull request, so I'll be putting it here (Sorry). One of the things I like about Yomichan is that you can upload your own dictionaries (In fact, it won't work if you don't), and while I can totally use Rikaichamp the way it is, I'd ratter use some J-J dictionary instead of the one used here. It could be a built-in option or a way to upload yourself, whatever is fine. By the way, I'd also ask for a "auto-start" option, if the button of the "press-to-translate" thing come to life. Thank you for the amazing work you've done here (y).

**Comment by birtles** [comment]
Yes, a J-J dictionary would be great. Do you know of a freely available one?

**Comment by birtles** [comment]
That sounds good. I think EPWING support is something worth looking into.

**Comment by birtles** [comment]
> By the way, I'd also ask for a "auto-start" option, if the button of the "press-to-translate" thing come to life. Thank you for the amazing work you've done here (y).

Press-to-translate is likely in the near future. Auto-start should already exist. If you have rikaichamp running when you close Firefox it should automatically re-start when you open Firefox again.

**Comment by birtles** [comment]
Updated title to scope this to EPWING support since auto-start is already implemented and press-to-translate is tracked elsewhere.

**Comment by picsi** [comment]
>Yes, a J-J dictionary would be great. Do you know of a freely available one?

There is https://ja.wiktionary.org/ but it's probably not very complete (and hard to scrape)

**Comment by nicolasmaia** [comment]
Might I suggest this be broadened to allow more dictionaries?

I say that because I think using Wikidata as a dictionary in Rikai would be extremely powerful.
